pom.xml

```diff
@@ -260,6 +262,39 @@
             <version>2.4</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <artifactId>configuration-as-code</artifactId>
+            <groupId>io.jenkins</groupId>
+            <version>1.36</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <artifactId>test-harness</artifactId>
+            <groupId>io.jenkins.configuration-as-code</groupId>
+            <version>1.36</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.main</groupId>
+            <artifactId>jenkins-test-harness</artifactId>
+            <version>2.56</version>
+        </dependency>
```

Comment:
Causes [JENKINS-65650](https://issues.jenkins.io/browse/JENKINS-65650).

Reply:
Also causes https://issues.jenkins.io/browse/JENKINS-65771 :(

Reply:
What a rookie mistake. Resoled in alpha in https://github.com/jenkinsci/gerrit-trigger-plugin/commit/63407f4c4a314043b39b21c5cc340bb9a8200e7b

Reply:
@rsandell, Can you please cut 2.35 (there is only a beta out). It affects unrelated plugins as Jenkins introspect this test-only dependency. Thanks!